Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -467,10 +488,10 @@
   "block.biomeswevegone.mossy_red_rock_brick_stairs": "Ступени из мшистого красного камня",
   "block.biomeswevegone.mossy_red_rock_brick_wall": "Стена из мшистого красного камня",
   "block.biomeswevegone.mossy_red_rock_bricks": "Мшистый красный камень",
+  "block.biomeswevegone.mossy_stone": "Мшистый камень",
   "block.biomeswevegone.mossy_stone_slab": "Плита из мшистого камня",
   "block.biomeswevegone.mossy_stone_stairs": "Ступени из мшистого камня",
   "block.biomeswevegone.mossy_stone_wall": "Стена из мшистого камня",
-  "block.biomeswevegone.mossy_stone": "Мшистый камень",
   "block.biomeswevegone.oddion_crop": "Урожай Луковички",
```

Лишняя заглавная буква
```suggestion
  "block.biomeswevegone.oddion_crop": "Урожай луковички",
```